packages/dx-react-grid/docs/reference/table-column-resizing.md

```diff
@@ -13,6 +13,7 @@ A plugin that manages table column widths.
 Name | Type | Default | Description
 -----|------|---------|------------
 columnWidths? | Array&lt;[TableColumnWidthInfo](#tablecolumnwidthinfo)&gt; | | Specifies column widths.
+minColumnWidth? | number | `45` for [Bootstrap3](https://www.npmjs.com/package/@devexpress/dx-react-grid-bootstrap3); `55` for [Bootstrap4](https://www.npmjs.com/package/@devexpress/dx-react-grid-bootstrap4); `40` for [Material UI](https://www.npmjs.com/package/@devexpress/dx-react-grid-material-ui); | Specifies min column width.
```

Comment:
I believe it's better to use 'minimal' instead of 'min' in the description.

Reply:
Should we mention units here? I mean `40px` etc.

Reply:
@SergeyAlexeev we didn't mention units before